Domain/Models/models.go

```
@@ -0,0 +1,10 @@
+package models
+
+type Person struct {
+	ID      string   `json:"uid" validate:"required"`
+	Name    string   `json:"name" validate:"required,min=2,max=100, pattern=^[a-zA-Z\s-]+$"`
```

golangci-lint

structtag: struct field tag `json:"name" validate:"required,min=2,max=100, pattern=^[a-zA-Z\s-]+$"` not compatible with reflect.StructTag.Get: bad syntax for struct tag value

(govet)